pkg/executor/infoschema_reader.go

```diff
@@ -1748,10 +1741,10 @@ func (e *memtableRetriever) setDataForMetricTables() {
 	e.rows = rows
 }
 
-func keyColumnUsageInTable(schema model.CIStr, table *model.TableInfo, extractor *plannercore.InfoSchemaKeyColumnUsageExtractor) [][]types.Datum {
+func keyColumnUsageInTable(schema model.CIStr, table *model.TableInfo, ex *plannercore.InfoSchemaKeyColumnUsageExtractor) [][]types.Datum {
 	var rows [][]types.Datum
 	if table.PKIsHandle {
```

Comment:
Merge L1746 and L1747

Reply:
https://github.com/pingcap/tidb/pull/55502/commits/569fc30d7aec000fa7a607c83db820d97cf0f4d3